scen_edit/command/save_command.lua

```diff
@@ -106,7 +106,7 @@ function SaveCommand.GenerateScript(dev)
     local modOptions = {
         deathmode = "neverend",
         has_scenario_file = not dev,
-        play_mode = not dev,
+        sb_gameMode = not dev and 2 or 0,
```

Comment:
This is starting to look unreadable.
Can we switch to using simple strings, as in "play" and "dev"?
I don't remember why I was using ints but it doesn't seem like a good idea anymore.